src/WebJobs.Script/Scale/TargetScalerVote.cs

```diff
@@ -0,0 +1,14 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Host.Scale;
+
+namespace Microsoft.Azure.WebJobs.Script.Scale
+{
+    internal class TargetScalerVote
```

Comment:
This will be used by the scale controller redesign directly, so this needs to be public (like ScaleVote)

Reply:
Ah, I see how this is used. Why have a separate class with the same attributes as `ScaleStatusResult`? If it is to purely distinguish which type of scaling algorithm is used, why not have a bool attribute `isTargetScaler` or something similar? `TargetScalerVote` appears redundant the way it's used right now

Reply:
> @alrod Could you please update the PR description with more context on changes this PR is bringing? This will help future readers and current reviewers like me who does not fully understand the scaling scenarios and impact of this change. Thank you!

@kshyju,
Added a link to the issue in the PR description.

Reply:
This s was based on @mathewc's comment:
https://github.com/Azure/azure-functions-host/pull/8780#discussion_r987194976

@chiangvincent, if `TargetWorkerCount` is set in the response SC is not going to take into account `Vote`, right?

Reply:
Yes, `TargetWorkerCount` takes precedence.

Reply:
`TargetScalerVote` is added to calculate correct value for `Vote` if there TBS is enabled. Even if SC does not use we are adding the logic for consistency in the response.